plugins/auth0/pkg/fetch/fetch.go

```diff
@@ -286,17 +297,13 @@ func (f *Fetcher) getUserRoles(ctx context.Context, uID string) ([]map[string]in
 	return results, nil
 }
 
-func (f *Fetcher) getOrgs(ctx context.Context, uID string) ([]map[string]interface{}, error) {
+func (f *Fetcher) fetchOrgs(ctx context.Context, uID string) ([]map[string]any, error) {
 	page := 0
 	finished := false
 
-	var results []map[string]interface{}
-
-	for {
-		if finished {
-			break
-		}
+	var results []map[string]any
 
+	for !finished {
```

Comment:
Here too, if you replaced `finished` with `hasNext` we wouldn't need to do all these boolean gymnastics.